package.json

```diff
@@ -45,7 +45,7 @@
     "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "prettier": "^3.2.2",
+    "prettier": "^3.2.4",
```

Comment:
This version seems to have a bug: It wants to add trailing commas to plain json files. AFAIC trailing comma in JSON is still not allowed. E.g. `JSON.parse('{"foo" : 1, }')` fails in JavaScript.

Reply:
See https://github.com/prettier/prettier/issues/15956 for Prettier issue.